### ISSUE TITLE
Published on Shinken.io

### ISSUE THREAD

**Issue by mohierf** [david-guenault/shinken-packs#5]
I do not think you published the last version of your Shinken2 pack on shinken.io ?

**Comment by david-guenault** [comment]
you are right ! lack of time and job priorities. As soon as possible and when i think everything is ok (btw do you have any idea of existing bugs or missing functionalities i should fix/implement ?)

**Comment by mohierf** [comment]
## First comment : my problems.

I met a problem with a fresh new Debian 7 ... I installed the most recent pack "by hand" and I got an exception when the checks are launched. After few investigations, I found that this line:

```
     statusresult = requests.get(uri, timeout=timeout, verify=False).json()
```

raises an uncatched exception!

I tried to understand and I found that calling the json() method directly seems to raise the exception.  There is alos a problem in line 72 : 

```
        if target in statusresult.keys():
```

 because .keys() does not exist !

I will propose a PR with some few modifications ...

**Comment by mohierf** [comment]
## Second comments : my ideas

As you may know, I started a lot of modifications in the WebUI and I would like to enrich some views : 
- system view 
  ![image](https://cloud.githubusercontent.com/assets/2932687/8205996/e887ac56-14f5-11e5-9ca5-ff3c43d2ec22.png)
- dashboard view : 
  ![image](https://cloud.githubusercontent.com/assets/2932687/8206010/fb5081c8-14f5-11e5-8cae-ac1a9d6aee3d.png)
  
  I am not sure that it will be possible to directly enrich those views ... but if you may include perfdatas in the check results it will be possible to include a graph from Graphite with those metrics. What do you think about it ?

**Comment by david-guenault** [comment]
nice catch for both problems. Submit pr and i will test it
for the ui i do not see wich type of metrics you want ? can you explain ?

2015-06-17 13:39 GMT+02:00 Frédéric MOHIER notifications@github.com:

**Comment by mohierf** [comment]
I do not know very much which data the shinken daemons are providing on request ... but perharps we could find a counter that will be usable as "Shinken activity" ? An idea, does the scheduler provides how many checks he launchezd since the last request ?

**Comment by david-guenault** [comment]
ok i see
well there are some performance metrics that will be really interesting to
load BUT there is also a module that does it on statsd. So it might be a
little bit redundant

2015-06-17 14:07 GMT+02:00 Frédéric MOHIER notifications@github.com:

**Comment by mohierf** [comment]
You are right ... the idea is not to create duplicates. But I do not know which data are provided by the daemons ... if you do not known of something interesting, we close this subject :-)

Ps: I just created a PR ...

**Comment by david-guenault** [comment]
getting metrics is simple, check what is provided by the api (ex:
http://host:port/api_full)

2015-06-17 14:15 GMT+02:00 Frédéric MOHIER notifications@github.com:

**Comment by mohierf** [comment]
Thanks, I will have a look, but still lacks some doc about it ... did not found anything anywhere :/P

**Comment by david-guenault** [comment]
will try to help a little

2015-06-17 18:42 GMT+02:00 Frédéric MOHIER notifications@github.com:
